data.sql

```diff
@@ -38,4 +38,27 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
 UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
 UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon');
 
--- Additional data modifications if needed
+-- Insert data into the visits table
+INSERT INTO visits (visit_id, animal_id, vet_id, visit_date)
```

Comment:
- [ ] [REQUIRED] Great job so far @Ochiengsteven , however, some part of the project requirement states that you should Insert the following data for vets:
- Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
- Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
- Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
- Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.

Insert the following data for specializations:
- Vet William Tatcher is specialized in Pokemon.
- Vet Stephanie Mendez is specialized in Digimon and Pokemon.
- Vet Jack Harkness is specialized in Digimon.